src/cmd/root.go

```diff
@@ -159,3 +161,37 @@ func isUnixSocketMode(cmd *cobra.Command) bool {
 		os.Getenv(config.EnvVarUnixSocketPath) != "" ||
 		cmd.Flags().Changed("unix-socket")
 }
+
+// refreshRateFlag is a custom flag type for the TUI's refresh rate.
+// It accepts both an integer in seconds and a duration string.
+type refreshRateFlag struct {
```

Comment:
I think this is worth doing once another flag has the same use case. For now, it's much better to just use a name that describes what the flag is for.

Also, `refreshRateFlag` exists to preserve backwards compatibility. We want programs that currently use `-r` to not break once this is merged, so we can't just use a `DurationVar` flag. Ideally, when adding new flags, we should just use `DurationVar` directly rather than accepting "either seconds in integer or duration in string".